tests/layer_tests/pytorch_tests/pytorch_layer_test_class.py

```diff
@@ -85,6 +85,9 @@ def _test(self, model, ref_net, kind, ie_device, precision, ir_version, infer_ti
 
         if not isinstance(fw_res, tuple):
             fw_res = (fw_res,)
+        
+        if inplace:
+            fw_res = tuple([fw_res[0], fw_res[0]])
```

Comment:
I think this shouldn't be correct way of handling inplace ops. The better way should be making model that has 2 outputs: x and y (if we consider activation like `y=relu(x)`)

Reply:
updated